DataFormats/TrackingRecHit/src/classes_def.xml

```diff
@@ -31,7 +31,7 @@
   <class name="edm::ClonePolicy<TrackingRecHit>"/> <!-- Root6 -->
   <class name="edm::Ref<edm::OwnVector<TrackingRecHit,edm::ClonePolicy<TrackingRecHit> >,TrackingRecHit,edm::refhelper::FindUsingAdvance<edm::OwnVector<TrackingRecHit,edm::ClonePolicy<TrackingRecHit> >,TrackingRecHit> >"/>
   <class name="edm::RefVectorIterator<edm::OwnVector<TrackingRecHit,edm::ClonePolicy<TrackingRecHit> >,TrackingRecHit,edm::refhelper::FindUsingAdvance<edm::OwnVector<TrackingRecHit,edm::ClonePolicy<TrackingRecHit> >,TrackingRecHit> >"/>
-  <class pattern="std::iterator<*edm::Ref*>"/>
+  <!--<class pattern="std::iterator<*edm::Ref*>"/>-->
```

Comment:
Warning: Unused class rule: std::iterator<*edm::Ref*>
config/SCRAM/GMake/Makefile.rules:1998: recipe for target 'tmp/osx1012_amd64_clang8/src/DataFormats/TrackingRecHit/src/DataFormatsTrackingRecHit/a/DataFormatsTrackingRecHit_xr.cc' failed
gmake: *** [tmp/osx1012_amd64_clang8/src/DataFormats/TrackingRecHit/src/DataFormatsTrackingRecHit/a/DataFormatsTrackingRecHit_xr.cc] Error 1
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2